src/TextureAsset.cpp

```diff
@@ -20,9 +20,10 @@ void TextureAsset::load(const std::string& file_path) {
     }
 
     //Load the texture into opengl here!
+    glEnable(GL_BLEND);
```

Comment:
This only happens on loads so really doesn't happen that often. RenderingSystem only makes GL calls per frame, so I think it's better here, I put it lower down for only RGBA assets though, since this only is useful for pictures with an alpha channel.